src/app/screens/Confirmation/index.tsx

```diff
@@ -1,20 +1,33 @@
 import React from 'react';
 import { View, Image } from 'react-native';
+import { useDispatch } from 'react-redux';
 import i18next from 'i18next';
 import CustomText from '@components/CustomText';
+import CustomButton from '@components/CustomButton';
 import icCheckSuccess from '@assets/icCheckSuccess.png';
 import icDanger from '@assets/icDanger.png';
-import { useRouteWithParams } from '@utils/navUtils';
+import actionCreators from '@redux/articles/actions';
+import { onResetStack, useNavigationWithParams, useRouteWithParams } from '@utils/navUtils';
 import Routes from '@constants/routes';
 
 import './i18n';
 import styles from './styles';
 import { ConfirmationTypes } from './constants';
 
 function Confirmation() {
+  const dispatch = useDispatch();
+  const navigation = useNavigationWithParams();
+
   const { type = ConfirmationTypes.SUCCESS_REGISTER_ARTICLE, typeError } =
     useRouteWithParams<Routes.Confirmation>().params || {};
 
+  const handlePress = () => {
+    setTimeout(async () => {
```

Comment:
Just out of curiosity, why is setTimeout used?

Reply:
fixed, was added on reload articles test